src/stores/AppStore.js

```diff
@@ -622,6 +625,18 @@ export default types
       }, 'Error during reject, try again');
     }
 
+    /**
+     * Exchange storage url for presigned url for task
+     */
+    async function presignUrlForProject(url) {
+      // Event invocation returns array of results for all handlers.
+      const urls = await getEnv(self).events.invoke('presignUrlForProject', self, url);
```

Comment:
let's use this `events` getter right away
```suggestion
      const urls = await self.events.invoke('presignUrlForProject', self, url);
```